README.md

```diff
@@ -200,6 +202,9 @@ DESCRIPTION
 EXAMPLES
   swaggerhub api:update organization/api --file api.yaml
   swaggerhub api:update organization/api/1.0.0 --file api.json
+  swaggerhub api:update organization/api/1.0.0 --publish --file api.json
+  swaggerhub api:update organization/api/1.0.0 --setdefault --file api.json
+  swaggerhub api:update organization/api/1.0.0 --publish --setdefault --file api.jso
```

Comment:
typo: `api.jso` => `api.json`

Reply:
Probably don't need examples of every combination of flags. I think just the last example is necessary to show the publish and setdefault flags.
i.e. The following lines could be removed:
``` 
swaggerhub api:update organization/api/1.0.0 --publish --file api.json
swaggerhub api:update organization/api/1.0.0 --setdefault --file api.json

Reply:
There is no harm if we keep them all. publish and setdefault are requiring file flag and it's good to show it in examples.